<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PetoiControllerQt v0.2.1a</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/app/pic/logo.icns</normaloff>:/app/pic/logo.icns</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>39</y>
      <width>981</width>
      <height>651</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tabSerial">
     <attribute name="title">
      <string>Serial</string>
     </attribute>
     <widget class="QGraphicsView" name="viewBittleIllustration">
      <property name="geometry">
       <rect>
        <x>579</x>
        <y>10</y>
        <width>391</width>
        <height>591</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBoxSerialPort">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>561</width>
        <height>291</height>
       </rect>
      </property>
      <property name="title">
       <string>Serial Connection</string>
      </property>
      <widget class="QSplitter" name="serialSetupSplitter">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>541</width>
         <height>221</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget_1">
        <layout class="QHBoxLayout" name="horizontalLayout_1">
         <item>
          <widget class="QLabel" name="labelSerialPort">
           <property name="text">
            <string>Port Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="boxPortNumber">
           <property name="minimumSize">
            <size>
             <width>360</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="labelBaudRate">
           <property name="text">
            <string>Baud Rate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="boxBaudRate">
           <property name="minimumSize">
            <size>
             <width>360</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_3">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="labelStopBits">
           <property name="text">
            <string>Stop Bits</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="boxStopBits">
           <property name="minimumSize">
            <size>
             <width>360</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_4">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="labelDataBits">
           <property name="text">
            <string>Data Bits</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="boxDataBits">
           <property name="minimumSize">
            <size>
             <width>360</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_5">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="labelParity">
           <property name="text">
            <string>Parity</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="boxParity">
           <property name="minimumSize">
            <size>
             <width>360</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QPushButton" name="buttonConnect">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>250</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonRefresh">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBoxServosAdjust">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>300</y>
        <width>561</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Calibration</string>
      </property>
      <widget class="QSplitter" name="splitter_2">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>70</y>
         <width>207</width>
         <height>32</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QPushButton" name="buttonCleanCalibration">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Clean</string>
        </property>
       </widget>
       <widget class="QPushButton" name="buttonSaveCalibration">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>541</width>
         <height>32</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QPushButton" name="buttonCalibration">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Calibration</string>
        </property>
       </widget>
       <widget class="QComboBox" name="boxSteering">
        <property name="minimumSize">
         <size>
          <width>270</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="buttonIncreaseAngle">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
       <widget class="QPushButton" name="buttonDecreaseAngle">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>10</y>
        <width>20</width>
        <height>620</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupTerminal">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>410</y>
        <width>561</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Terminal</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>541</width>
         <height>161</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QSplitter" name="splitter_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QLineEdit" name="lineCmd">
           <property name="minimumSize">
            <size>
             <width>240</width>
             <height>0</height>
            </size>
           </property>
          </widget>
          <widget class="QPushButton" name="buttonSendCmd">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Send</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="textTerminalOutput"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tabPanel">
     <attribute name="title">
      <string>Actions</string>
     </attribute>
     <widget class="QGroupBox" name="groupBoxActions">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>631</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Default Actions</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>31</y>
         <width>611</width>
         <height>160</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayoutActions">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="buttonPosStretch">
            <property name="text">
             <string>Stretch</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonPosButtomUp">
            <property name="text">
             <string>Hip Up</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonPosCheckRd">
            <property name="text">
             <string>Look Around</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="buttonPosHi">
            <property name="text">
             <string>Greeting</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonPosPee">
            <property name="text">
             <string>Pee</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonPosPushUp">
            <property name="text">
             <string>Pushup</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QPushButton" name="buttonPosSit">
            <property name="text">
             <string>Sit Down</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonPosSleep">
            <property name="text">
             <string>Sleep</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonPosRest">
            <property name="text">
             <string>Rest</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QPushButton" name="buttonPosStepping">
            <property name="text">
             <string>Stepping</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonPosBackFlip">
            <property name="text">
             <string>Back Flip</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonPosBunnyJump">
            <property name="text">
             <string>Bunny Jump</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBoxServosPanel">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>10</y>
        <width>311</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Bittle Control</string>
      </property>
      <widget class="QTextBrowser" name="textSpecification">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>291</width>
         <height>161</height>
        </rect>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Self-right (recovery)	R&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Gyro on/off		G	&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Servos on/off	H	 &lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Up: 	W 	Normal: 	1&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Down:	S 	Run: 	2&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Left:	A	Crawl: 	3&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Right:	D&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBoxCustomActions">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>951</width>
        <height>391</height>
       </rect>
      </property>
      <property name="title">
       <string>Custom Commands</string>
      </property>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>721</width>
         <height>351</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>770</x>
         <y>30</y>
         <width>171</width>
         <height>100</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>New</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QSplitter" name="languageSplitter">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>10</y>
      <width>221</width>
      <height>32</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="labelLanguage">
     <property name="text">
      <string>Language</string>
     </property>
    </widget>
    <widget class="QComboBox" name="boxLanguage"/>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
